### ISSUE TITLE
Unable to give Dr. Anya gifts

### ISSUE THREAD

**Issue by TerryFoldsPrime** [itsamewayu/OLFallenDoll#2096]
Describe the bug
When attempting to give Dr. Anya a gift toward Level 6: Lover affinity, the gift button does not activate.

To Reproduce
1. Go to Harem Mode
2. Click on Character
3. Select Dr. Anya
4. Click on the Affinity Tab
5. Select a gift
6. Notice the Gift button does not light up
7. 
Expected behavior
Selecting a gift and pressing the gift button increases the affinity level.

Screenshots
![20241024182100_1](https://github.com/user-attachments/assets/986c4014-eaef-4c51-8f73-2232d00dc496)

Desktop (please complete the following information):
 - OS: Win10
 - Game Version 0.8.5

**Comment by Ageofwar666** [comment]
I'm having the same issue with Alet. So I think the issue is giving gifts to Investagtors in general

**Comment by yiwu0b11** [comment]
try to give gifts to someone below level5. IRRC, lvl6 is not implemented in game, so you can't unlock it.

**Comment by Ageofwar666** [comment]
So anyone that is not on the Wedding dress stage?

**Comment by TerryFoldsPrime** [comment]
Yeah, I don't have any issues with any other investigator below level 6. That's very disappointing that it has not been implemented - I was very excited.
